src/Components/WebAssembly/Server/src/ComponentsWebAssemblyApplicationBuilderExtensions.cs

```diff
@@ -101,7 +101,7 @@ private static StaticFileOptions CreateStaticFilesOptions(IFileProvider webRootF
                     // When we revisit this, we should consider calculating the original content type and storing it
                     // in the request along with the original target path so that we don't have to calculate it here.
                     var originalPath = Path.GetFileNameWithoutExtension(requestPath.Value);
-                    if (contentTypeProvider.TryGetContentType(originalPath, out var originalContentType))
+                    if (originalPath != null && contentTypeProvider.TryGetContentType(originalPath, out var originalContentType))
```

Comment:
originalPath should never be null here right?  But I guess this is needed for nullability assertions?

Reply:
`GetFileNameWithoutExtension` returns a nullable string.